.circleci/config.yml

```
@@ -174,6 +187,18 @@ jobs:
           command: |
             yarn test:unit:web:ci
 
+  test-web:
+    docker:
+      - image: cimg/node:12.16
+    working_directory: ~/pass-culture
+    steps:
+      - checkout
+      - install_node_modules
+      - run:
+          name: Unit Tests Web
+          command: |
```

```suggestion
          command: yarn test:unit:web:ci
```

tu peux les faire tenir sur une ligne si y'a qu'une commande